web/src/components/tasks/LicenseTask.test.tsx

```diff
@@ -462,15 +459,57 @@ describe("<LicenseTask />", () => {
         renderTask();
         fireEvent.click(screen.getByText(Config.licenseSearchTask.tab2Text));
         expect(screen.queryByTestId("error-alert-NOT_FOUND")).not.toBeInTheDocument();
+        mockApi.checkLicenseStatus.mockResolvedValue(
```

Comment:
[pebble] How does the `checkLicenseStatus` function return an empty string? The empty object is returned on line 96 in `updateLicenseStatusFactory` and that uses `updateLicenseStatusAndLicenseTask` function which sets `lastUpdatedISO`to currentDate on line 58? Let me know if I'm getting this confused.